examples/reminders/lib/base/di/app_dependencies.dart

```diff
@@ -160,4 +195,28 @@ class _AppDependenciesState extends State<AppDependencies> {
           create: (context) => AnalyticsInterceptor(context.read()),
         ),
       ];
+
+  Future<void> initRealm() async {
```

Comment:
Do we really need to have the real initialization as part of the app_dependencies ? I believe we can implement it in more scalable manner so that if the number of storages raises the app_dependencies do not get overloaded